source/sass/mixins/_logical-properties.scss

```diff
@@ -45,19 +47,26 @@
   }
 }
 
+@function _get-property-prefix($property-name, $dimension) {
```

Comment:
Is this needed anymore?

Reply:
Recommend considering this as part of https://github.com/libero/libero/issues/180.